### ISSUE TITLE
[Bug] UI version 3.8 lists Ubuntu 18.04 but YAML config validator complains it is not supported

### ISSUE THREAD

**Issue by paragao** [aws/aws-parallelcluster-ui#318]
## Description
The ParallelCluster UI (PCUI) version 2023.12.0 shows a choice of Ubuntu 18.04 as the operating system for the cluster when creating a new cluster. When you try to create it, the YAML validator complains that Ubuntu 18.04 is not supported and the only supported options are 2004 and 2204. Ubuntu 20.04 is an option when creating a new cluster but 22.04 is not.

## Steps to reproduce the issue
1. Install AWS ParallelCluster 3.8.0 and PCUI 2023.12.0
2. Click on "Create cluster" and choose "Step by step".
3. On Step 1, "Cluster", under Operating System, choose Ubuntu 18.04
4. You can fill the other options and steps with any information you want
5. At Step 5, "Create", click on the "Create" button. The YAML config validator will complain that ubuntu1804 is not a supported option. 

## Expected behaviour
When creating a new cluster, to find only supported operating system options on Step 1 and be able to create the cluster without an error.

## Actual behaviour
Instead of being able to create a cluster, I got an error message saying the operating system chosen was not supported. The error message gave me two supported options but one of them is not available during the cluster creation process. You can find the screenshots attached.
![Screenshot 2024-02-19 at 14 42 12](https://github.com/aws/aws-parallelcluster-ui/assets/5193870/9c37ace7-0f07-4c04-865b-c08abb515ba7)
![Screenshot 2024-02-19 at 14 42 37](https://github.com/aws/aws-parallelcluster-ui/assets/5193870/fe5b11b7-d8f8-4d59-ae65-1fafb2428ac2)

## Required info
In order to help us determine the root cause of the issue, please provide the following information:
- Region where ParallelCluster UI is installed: us-east-2
- Version of ParallelCluster UI and ParallelCluster: 3.8.0 and 2023.12.0 respectively
- Logs: N/A

## Additional info
The following information is not required but helpful:
 - OS: MacOS 13.6.3
 - Browser: tried Firefox 15.7.0esr, Chrome 120.0.6099.129 and Safari 17.3.1. Same behavior on all of them.

## If having problems with cluster creation or update
Cluster YAML (used ??? as a token to replace confidential data)
```
HeadNode:
  InstanceType: m6a.large
  Imds:
    Secured: true
  Ssh:
    KeyName: ???
  LocalStorage:
    RootVolume:
      VolumeType: gp3
      Size: 40
  Networking:
    SubnetId: subnet-???
  Iam:
    AdditionalIamPolicies:
      - Policy: arn:aws:iam::aws:policy/AmazonSSMManagedInstanceCore
Scheduling:
  Scheduler: slurm
  SlurmQueues:
    - Name: genai
      AllocationStrategy: lowest-price
      ComputeResources:
        - Name: genai-cr-0
          Instances:
            - InstanceType: c6a.2xlarge
          MinCount: 0
          MaxCount: 16
          DisableSimultaneousMultithreading: true
      ComputeSettings:
        LocalStorage:
          RootVolume:
            VolumeType: gp3
            Size: 40
      Networking:
        SubnetIds:
          - subnet-???
        PlacementGroup: {}
  SlurmSettings:
    QueueUpdateStrategy: COMPUTE_FLEET_STOP
    ScaledownIdletime: 5
Region: us-east-2
Image:
  Os: ubuntu2004
Imds:
  ImdsSupport: v2.0
DirectoryService:
  GenerateSshKeysForUsers: true
  DomainReadOnlyUser: CN=???,OU=???,OU=???,DC=???,DC=???,DC=com
  DomainAddr: ldap://???.???.com
  DomainName: dc=???,dc=???,dc=com
  PasswordSecretArn: >-
    arn:aws:secretsmanager:???:???:secret:???
Tags:
  - Key: parallelcluster-ui
    Value: 'true'
```

## If having problems with custom image creation
N/A

**Comment by judysng** [comment]
Hi @paragao,

Thanks for bringing up the issue. I have addressed this UI issue in [this PR](https://github.com/aws/aws-parallelcluster-ui/pull/320) for the next release of PCUI. ParallelCluster v3.7 deprecated Ubuntu18.04, so if you would like to use Ubuntu18.04 you can use an older version of pcluster/PCUI.

**Comment by gmarciani** [comment]
This issue has been fixed in PCUI 2024.03.0
https://github.com/aws/aws-parallelcluster-ui/releases/tag/2024.03.0
